quesma/quesma/schema_transformer.go

```diff
@@ -845,6 +845,41 @@ func (s *SchemaCheckPass) checkAggOverUnsupportedType(indexSchema schema.Schema,
 
 }
 
+func columnsToAliasedColumns(columns []model.Expr) []model.Expr {
+	aliasedColumns := make([]model.Expr, len(columns))
+	for i, column := range columns {
+		if columnRef, ok := column.(model.ColumnRef); ok {
+			aliasedColumns[i] = columnRef
+			continue
+		}
+		if col, ok := column.(model.LiteralExpr); ok {
+			if _, isStr := col.Value.(string); !isStr {
+				aliasedColumns[i] = model.NewAliasedExpr(column, fmt.Sprintf("column_%d", i))
+			} else {
+				aliasedColumns[i] = col
+			}
+			continue
+		}
+		if aliasedExpr, ok := column.(model.AliasedExpr); ok {
+			aliasedColumns[i] = aliasedExpr
+			continue
+		}
+		if _, ok := column.(model.FunctionExpr); ok {
```

Comment:
This is tricky. If we have a query like "select column, count(*) from .."  the database will name the second column as 'count'. This code changes the name to 'column_1'.  Not sure how it affects the results rendering. 

Generally speaking, we should name every column on the parse step.

Reply:
`the database will name the second column as 'count'. This code changes the name to 'column_1'.`

The existing code in `ProcessQuery` would represent unaliased `count` as `column_1` in `QueryResultRow`, so this naming is compatible with the existing code.